util/test/perf/perfgraph.js

```diff
@@ -732,7 +778,8 @@ function customValueFormatter(val, opts, series_name, dygraph) {
 
   // update digits, but do NOT redraw. Then use the default value formatter
   dygraph.updateOptions({digitsAfterDecimal: digits}, true);
-  return Dygraph.numberValueFormatter(val, opts);
+
+  return val.toFixed(digits);
```

Comment:
Hmm, I had forgotten about the scientific notation switch. The ``toExponential`` method may be useful. Do you have an example graph in mind?